enricher/src/main/kotlin/events/boudicca/enricher/model/Event.kt

```diff
@@ -0,0 +1,9 @@
+package events.boudicca.enricher.model
+
+import java.time.ZonedDateTime
+
+data class Event(
```

Comment:
we really should not duplicate this all over

Reply:
we should not, but with the current openapi setup we have to do it, because openapi is, once again, a pain...

Reply:
yes, we should really take care of the openAPI stuff once and for all...